### PR TITLE
Remove DNNCorp copyright

### LINKED ISSUE #9: Make copyright in site templates non DNNCorp
Change the copyright

### PR DESCRIPTION
**Pull request by EPTamminga**
Close #9

### CODE REVIEW

File: Resources/Portals/_default/Blank Website.template.fr-FR.resx [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <value xml:space="preserve">fr-FR</value>
   </data>
   <data name="Setting.FooterText.Text">
-    <value xml:space="preserve">Copyright [year] DNN Corp</value>
+    <value xml:space="preserve">Copyright [year] mon site internet</value>
```
Comment (valadas) [inline Resources/Portals/_default/Blank Website.template.fr-FR.resx]:
```suggestion
    <value xml:space="preserve">Copyright [year] mon site web</value>
```

File: Resources/Portals/_default/Default Website.template.fr-FR.resx [hunk 2]
```diff
@@ -130,7 +130,7 @@
     <value xml:space="preserve">fr-FR</value>
   </data>
   <data name="Setting.FooterText.Text" xml:space="preserve">
-    <value xml:space="preserve">Copyright [year] DNN Corp</value>
+    <value xml:space="preserve">Copyright [year] mon site internet</value>
```
Comment (valadas) [inline Resources/Portals/_default/Default Website.template.fr-FR.resx]:
```suggestion
    <value xml:space="preserve">Copyright [year] mon site web</value>
```

**Review by valadas — CHANGES_REQUESTED** [review]
I would prefer this wording

**Comment by EPTamminga** [comment]
@valadas _Your wish is my command. Consider it done_.

File: Resources/Portals/_default/Default Website.template.fr-FR.resx (hunk 2 above)
Comment (EPTamminga) [reply Resources/Portals/_default/Default Website.template.fr-FR.resx]: Done

**Comment by valadas** [comment]
Thank you sir!

**Review by valadas — APPROVED** [review]

**Pull request merged**